app/javascript/components/ProductEdit/ProductTab/CoverEditor.tsx

```diff
@@ -154,51 +156,58 @@ const CoverUploader = ({
       <LoadingSpinner className="size-20" />
     ) : (
       <div style={{ width: "100%" }}>
-        <div className="tab-buttons small" role="tablist">
-          <label className="button" role="tab">
-            <input
-              type="file"
-              multiple
-              accept={ALLOWED_EXTENSIONS.map((ext) => `.${ext}`).join(",")}
-              disabled={isUploading}
-              onChange={asyncVoid(async (event) => {
-                if (!event.target.files?.length) return;
+        <Tabs variant="buttons">
```

Comment:
The `small` style was only used here (and technically it was bigger...), so I just kept the default as it actually makes it more compact. Lmk if you disagree.